{
  "bug_location": "Unknown",
  "severity": 3,
  "categories": [
    "Unknown"
  ],
  "root_cause": "N/A - Analysis failed",
  "issue_number": 8506,
  "title": "cases::test_merge::test_node_mutiple_rollback_merge failed"
}